share/spice/similar_sites/similar_sites.css

```diff
@@ -1,20 +1,16 @@
-.zci--similar_sites #hidden {
+.zci--similar_sites .hidden, .zci--similar_sites .chomp--link__ls {
```

Comment:
sorry, our class name is actually, `is-hidden` -- you should be able to use that.